monocle/sanitized.py

```diff
@@ -168,6 +169,7 @@
     'LOAD_CUSTOM_JS_FILE': False,
     'LOGIN_TIMEOUT': 2.5,
     'MANAGER_ADDRESS': None,
+    'MAP_FILTER_IDS': False,
```

Comment:
I would prefer for this to be `None`. I just use `False` for booleans and `None` for the absence of an option that would have another type (a sequence in this case). They both fail the truth test.